io.github.ctlcltd.e2se.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.ctlcltd.e2se
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: e2-sat-editor
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=cups
+  - --filesystem=host
+rename-desktop-file: e2-sat-editor.desktop
+rename-icon: e2-sat-editor
+cleanup:
+  - '/include'
+  - '/bin/curl-config'
+  - '/lib/cmake'
+  - '/lib/debug'
+  - '/lib/pkgconfig'
+modules:
+  - name: e2-sat-editor
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DWITHOUT_CLI=ON
+    subdir: src
+    sources:
+      - type: git
+        url: https://github.com/ctlcltd/e2-sat-editor.git
+        # branch: main
+        tag: v0.7.0
+        commit: 801dc65d3a5d9fbb5a2b0d75bb2792ac2ce4b98b
+    modules:
+      - name: libcurl4
```

Comment:
curl is in the runtime. why do you need to build it?

Reply:
It is not documented that **libcurl4** is included. Seems to work without the package. I don't know if you have a reference to Platform/Sdk used where it is said to provide **libcurl4**.

Reply:
how about you remove it then

Reply:
I checked org.kde.Sdk and org.kde.Platform at version 6.4, they contain curl 7.8.6 compiled from the curl repo on github, no additional patches or build options (manifest-base-1.json). The libraries contained in the kde runtime are not listed anywhere, could they be removed in the near future? I don't know, it works without for now. I take off